lib/govuk_app_config/xray.rb

```diff
@@ -0,0 +1,12 @@
+require 'aws-xray-sdk/facets/rails/railtie'
+
+Rails.application.config.xray = {
+  name: ENV["GOVUK_APP_NAME"].to_s,
+  patch: %I[net_http aws_sdk],
+  sampling_rules: {
+    default: {
+      "fixed_target": ENV.fetch("GOVUK_XRAY_SAMPLE_TARGET", 0).to_i,
```

Comment:
I don't think we need the `GOVUK_` prefix on these, and I wonder if we'll need to cast them to floats before setting the argument?

Reply:
Casting beyond the current `.to_i` and `.to_f`?

Reply:
Not sure how I missed that...